wire-ios/Wire-iOS.xcodeproj/xcshareddata/xcschemes/Wire-iOS.xcscheme

```diff
@@ -233,14 +233,6 @@
             argument = "--persist-backend-type"
             isEnabled = "NO">
          </CommandLineArgument>
-         <CommandLineArgument
-            argument = "-UseAnalytics 1"
-            isEnabled = "NO">
-         </CommandLineArgument>
-         <CommandLineArgument
-            argument = "-ConsoleAnalytics"
-            isEnabled = "NO">
```

Comment:
Do you know what these were for? I'm wondering if it was to prevent sending analytics data when running tests.

Reply:
Even if that's the case, we check first and we're not setting up an analytics session when running tests.